events/app.py

```diff
@@ -72,7 +72,31 @@ def edit_event(event_id):
 @app.route('/v1/', methods=['GET'])
 def get_all_events():
     """Return a list of all events currently in the DB."""
-    pass
+    try:
+        events = events_collection.find()
+        events_dict = build_events_dict(events)
+        # TODO(cmei4444): test with pageserve to make sure the json format is
+        # correct in the response
+        return Response(
+            events_dict,
+            status=200,
+            mimetype='application/json'
+        )
+    except DBNotConnectedError as e:
+        return Response(
+            status=500,
+            response="Database was undefined.",
+        )
```

Comment:
In flask you can also do
```python
return "Database was undefined.", 500
```
Which may be a bit cleaner. The documented use of the `Response` constructor seems to be if you need to do something fancy like changing headers, but you aren't yet so you can rely on implicit response creation.